### PR TITLE
Use n2-highcpu-32 for DeepSpeech

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

**Comment by lissyx** [comment]
Also, I don't yet have setup `tc-admin`, can this be deployed for me?

**Comment by SimonSapin** [comment]
@lissyx Note that `n2-*` machine are only available in some GCP regions / zones, and even then availability is limited. For Servo I’ve switched to `n1-*`, though I haven’t looked much into the details.

**Comment by lissyx** [comment]
Good to know, but the variability **and** the increase in build time is really painful :/

**Comment by djmitche** [comment]
I think they're in only one region, and only 24 CPUs allowed per project.  Let's look at this next week -- we may be able to request a limit increase from Google.

**Comment by lissyx** [comment]
Duh, that's a lot of limitations all of a sudden...

**Comment by djmitche** [comment]
Google only very recently announced the n2's.  I think they're still racking them up..

**Comment by djmitche** [comment]
@ccooper has more data, i think..

File: config/projects.yml [hunk 1]
```diff
@@ -443,7 +443,7 @@ deepspeech:
       type: standard_gcp_docker_worker
       minCapacity: 0
       maxCapacity: 96
-      machineType: "zones/{zone}/machineTypes/n1-standard-32"
+      machineType: "zones/{zone}/machineTypes/n2-highcpu-32"
```
Comment (ccooper) [inline config/projects.yml]: n2s are available now, but only in certain regions and zones: https://cloud.google.com/compute/docs/regions-zones/#available

You could potentially use them, but you'd need to be explicit about the zone(s) in your patch.

Comment (lissyx) [reply config/projects.yml]: Thanks! That page is hard to decipher :[

**Comment by djmitche** [comment]
Looks like this is stalled out?

**Comment by lissyx** [comment]
it did not, I was waiting to let the dust settle and have some more feedback on this zone thing.

**Comment by lissyx** [comment]
So far we don't need this anymore for the moment.

**Pull request closed without merging**